check.go

```diff
@@ -124,8 +122,17 @@ func (d *DBFT[H]) checkCommit() {
 		zap.Stringer("merkle", d.block.MerkleRoot()),
 		zap.Stringer("prev", d.block.PrevHash()))
 
+	err := d.ProcessBlock(d.block)
+	if err != nil && d.isAntiMEVExtensionEnabled() { // errors not supported for AntiMEVExtensions mode off.
```

Comment:
That's not obvious for library users, we need to handle them somehow anyway.

Reply:
Fixed.